roles/ovirt-datacenter-cleanup/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+- name: Remove VMPools
+  include: vm_pools.yml
```

Comment:
In Ansible 2.4 there is now dynamic include and static include:

 - http://docs.ansible.com/ansible/2.4/include_tasks_module.html
 - https://docs.ansible.com/ansible/2.4/import_tasks_module.html

`include` is depracated and will be removed in future. Please change it if you think you don't need this to work with Ansible < 2.4. I will merge for now.